test/v2/load/load_generator_metrics.go

```diff
@@ -48,7 +54,7 @@ func newLoadGeneratorMetrics(registry *prometheus.Registry) *loadGeneratorMetric
 		[]string{},
 	)
 
-	relayReadSuccesses := promauto.With(registry).NewCounterVec(
+	relayReadSuccesses := promauto.With(registerer).NewCounterVec(
```

Comment:
nit: internet says registrar is more common

Reply:
I'm just following the prometheus defined interface:
![image](https://github.com/user-attachments/assets/fff0da3a-ad22-4465-a777-211a4a7f9056)